widgets/src/c3/Common.js

```diff
@@ -3,7 +3,7 @@
     if (typeof define === "function" && define.amd) {
         define(["d3/d3", "c3/c3", "../common/HTMLWidget", "css!c3/c3"], factory);
     } else {
-        root.Pie = factory(root.d3, root.c3, root.HTMLWidget);
+        root.C3_Common = factory(root.d3, root.c3, root.HTMLWidget);
```

Comment:
That's an old commit. The most recent commit should be identical to what I have in https://github.com/hmsonline/Visualization. Everything was changed lower case.